CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# CONTRIBUTING
+
+Contributions to the XRPL-Standards repo are free, open source, and follow the following document publishing process.
+
+Any XRPL-Standards document can be referred to interchangeably as an "XLS" (XRPL Standard), "RFC" (Request for Comments), or "document".
+
+## Summary
+
+Copyright on all content is subject to the terms of the [MIT License](LICENSE).
+
+All contributors grant everyone:
+
+Copyright – a royalty-free license to use the copyrights for their contributions.
+Patent – a commitment to license on a royalty-free basis their essential patent claims reading on their contributions.
+
+## Background
+
+The work of the XRP Ledger community is open source, collaborative, and welcoming of all contributors participating in good faith. [Learn more about the XRP Ledger at XRPL.org](https://xrpl.org/).
+
+This XRPL-Standards process is influenced by the well-established [IETF RFC process](https://www.ietf.org/standards/process/informal/).
+
+## Process
+
+The XRPL-Standards process attempts to be easy to use, but also rigorous enough that there are permalinks to revisions of documents for reference purposes.
+
+### Gathering Feedback Before Submitting
+
+Please gather community input before opening a PR. Collecting such feedback may be helpful to refine your concept.
```

Comment:
One point of clarification possibly between what this spec is suggesting and maybe what you're preferring @Silkjaer -- the original intent of this PR was actually to suggest that there should be a minimal bar to getting a PR into a draft mode (but the current two categories in this PR maybe don't allow for that).

The motivation for "low barrier" is for spec editing. It's really difficult to spec-edit in a GH discussion, so while having discussions in there is fine, it's important to create low barriers to initial PR so that a spec can be efficiently improved/edited. You can see that in action here in _this_ PR between you and me -- imagine having this back and forth in GH discussions!

I suppose what I'm getting at is it should be fine for us to have many, poor quality "draft" specs -- that means we have lots of things being proposed, and we just need a mechanism to distinguish between "random ideas" and "specs with legs" and "things that are final"

To that end, perhaps we should introduce a third category in this PR, maybe like this:

1. **Proposal**: A Proposal is an RFC document that proposes some new feature, protocol or idea (i.e., any random idea, low-barrier to PR).
2. **Working Draft**: A Working Draft is an RFC that has become stable enough for people to experiment with, but has not necessarily been endorsed by the entire community.
3. **Candidate Specification**: A Candidate Specification is a document that was previously a Working Draft RFC that is considered stable enough by the community such that no further changes are required.

Reply:
I would be fine with introducing a third category in this PR if that's what @Silkjaer wants. But until @Silkjaer confirms, I don't think it's necessary.

Per @Silkjaer 's request, I added a line here to make it clear that gathering community feedback is required.